File: vol1/ch6/LeetCode94-1.cpp
class Solution {
    vector<int> ans;
public:
    vector<int> inorderTraversal(TreeNode* root) {
        inorder(root);
        return ans;
    }

    void inorder(TreeNode* root) {					//µÝ¹éÖÐÐò±éÀúËã·¨
        if(root!=NULL) {
            inorder(root->left);
            ans.push_back(root->val);
            inorder(root->right);
        }
    }
};

File: vol1/ch6/LeetCode94-1.py
class Solution:
    def inorderTraversal(self, root: Optional[TreeNode]) -> List[int]:
        self.ans=[]
        self.inorder(root)
        return self.ans

    def inorder(self,root):							#递归中序遍历算法
        if root!=None:
            self.inorder(root.left)
            self.ans.append(root.val)
            self.inorder(root.right)
